ssl/ssl_lib.c

```diff
@@ -2506,6 +2506,24 @@ STACK_OF(SSL_CIPHER) *SSL_CTX_get_ciphers(const SSL_CTX *ctx)
     return NULL;
 }
 
+/* Distinguish between ciphers controlled by set_ciphersuite() and
```

Comment:
Fix comment; text should start on the line after the start of a multi-line comment. Please review the coding/formatting standards.